general/documentation/forking-versions.md

```diff
@@ -0,0 +1,42 @@
+---
+title: Managing Moodle version documentation
+sidebar_label: Moodle versions
+tags:
+  - documentation
+  - forking
+  - versions
+---
+
+We generate version-specific documentation and maintain this for each supported version of Moodle.
+
+When a new version of Moodle is released, the documentation is 'forked'.
+
+When a version of Moodle goes out of support, its developer documentation is archived.
+
+Further information about this functionality is available in the [Docusaurus documentation on the topic](https://docusaurus.io/docs/versioning).
+
+## Creating a new version of the docs
+
+Typically this task is performed by the Integration team using the following steps:
+
+1. Ensure that all pending appropriate merge requests have been merged
+1. Ensure that your local branch is up-to-date
+1. Run the docusaurus version command:
+
+    ```bash
+    yarn docusaurus docs:version [version]
+    ```
+
+1. Commit the initial changes ([Example from Moodle 4.4](https://github.com/moodle/devdocs/commit/e9e7fa0074753487c315d2f91ad64a8503f32054))
+1. Open `versioned_docs/version-[version]/intro.md` in your editor
+1. Uncomment and update the link to the release notes for this version
+1. Open `docs/devupdate.md` in your editor
+1. Clear the content of this file and update the version numbers
+1. Open `nextVersion.js` in your editor
+1. Update the values for `nextVersion` and `nextLTSVersion`
```

Comment:
It's super minor, but it's probably worth mentioning to only update `nextLTSVersion` when necessary. For example:

> Update the value for `nextVersion`. If it's an LTS release, update the `nextLTSVersion` as well.

Or something to that effect.

Reply:
> Update the values for `nextVersion` (and `nextLTSVersion` after the release of an LTS version)